src/ShallowWrapper.js

```diff
@@ -254,6 +254,15 @@ export default class ShallowWrapper {
   }
 
   /**
+   * Returns true if the component rendered nothing, i.e., null or undefined.
+   *
+   * @returns {boolean}
+   */
+  isNull() {
+    return (this.node === null);
```

Comment:
please remove the unnecessary parens.

if this is the canonical location for "is null", we should call into `isNull` in a number of other places, like `.type()` for example.